secant/Features/Onboarding/OnboardingStore.swift

```diff
@@ -9,20 +9,22 @@ import Foundation
 import SwiftUI
 import ComposableArchitecture
 
-struct OnboardingStep: Equatable, Identifiable {
-    let id: UUID
-    let title: String
-    let description: String
-}
-
 struct OnboardingState: Equatable {
-    var steps: IdentifiedArrayOf<OnboardingStep> = Self.onboardingSteps
+    struct Step: Equatable, Identifiable {
+        let id: UUID
+        let title: String
+        let description: String
+        let background: Image
+        let badge: Badge
+    }
+
+    var steps: IdentifiedArrayOf<Step> = Self.onboardingSteps
     var index = 0
     var skippedAtindex: Int?
     
-    var currentStep: OnboardingStep { steps[index] }
-    var skipButtonDisabled: Bool { steps.count == index + 1 }
-    var backButtonDisabled: Bool { index == 0 }
+    var currentStep: Step { steps[index] }
+    var finalStep: Bool { steps.count == index + 1 }
+    var initialStep: Bool { index == 0 }
```

Comment:
[non-blocking | naming | suggestion]

What do you think about prepending these properties with "is"?

```suggestion
    var isFinalStep: Bool { steps.count == index + 1 }
    var isInitialStep: Bool { index == 0 }
```